integrated_channels/moodle/migrations/0001_initial.py

```diff
@@ -1,22 +1,20 @@
-# Generated by Django 2.2.15 on 2020-08-20 16:06
+# Generated by Django 2.2.14 on 2020-08-27 20:11
```

Comment:
look at you using an old version of Django. :p

Reply:
This is with a brand new install in my venv too!